src/debugger/direct/directDebugAdapter.ts

```diff
@@ -69,25 +69,29 @@ export class DirectDebugAdapter extends ChromeDebugAdapter {
 
         return new Promise<void>((resolve, reject) => this.initializeSettings(launchArgs)
             .then(() => {
-                this.outputLogger("Launching the app");
-                logger.verbose(`Launching the app: ${JSON.stringify(launchArgs, null , 2)}`);
-                return TelemetryHelper.generate("launch", extProps, (generator) => {
-                    return this.remoteExtension.launch({ "arguments": launchArgs })
-                        .then(() => {
-                            return this.remoteExtension.getPackagerPort(launchArgs.cwd);
+                this.outputLogger("Launching the application");
+                logger.verbose(`Launching the application: ${JSON.stringify(launchArgs, null , 2)}`);
+                return ReactNativeProjectHelper.getReactNativeVersionFromProjectPackage(launchArgs.cwd)
+                    .then(version => {
+                        TelemetryHelper.addReactNativeVersionToEventProperties(version, extProps);
```

Comment:
Why we don't need to rewrite extProps object after that?

Reply:
Can we use `Object.assign` for that?
Example:
```
extProps = Object.assign({}, extProps, version);
```

Reply:
We can use the following:
```
extProps = Object.assign({}, extProps, {value: version, isPii: false});
```
But we already have `addReactNativeVersionToEventProperties`function  in `TelemetryHelper` class, which does almost the same as code above. I consider that usage of that function can be more appropriate

Reply:
Fixed.